test/controllers/ingredient_items_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class IngredientItemsControllerTest < ActionDispatch::IntegrationTest
+  test "should get new" do
+    get ingredient_items_new_url
+    assert_response :success
+  end
```

Comment:
You can remove this or change it to test the create method, since there is no `new` route